llvm/test/MC/X86/tls.s

```diff
@@ -0,0 +1,40 @@
+// RUN: llvm-mc -filetype=obj -triple=x86_64 %s -o %t
+// RUN: llvm-objdump -dr --no-print-imm-hex %t | FileCheck %s
+
+// CHECK:      <_start>:
+// CHECK-NEXT:   movq (%rip), %r16
+// CHECK-NEXT:   R_X86_64_CODE_4_GOTTPOFF    tls0-0x4
+// CHECK-NEXT:   movq (%rip), %r20
+// CHECK-NEXT:   R_X86_64_CODE_4_GOTTPOFF    tls0-0x4
+// CHECK-NEXT:   movq (%rip), %r16
+// CHECK-NEXT:   R_X86_64_CODE_4_GOTTPOFF    tls1-0x4
+// CHECK-NEXT:   addq (%rip), %r16
+// CHECK-NEXT:   R_X86_64_CODE_4_GOTTPOFF    tls0-0x4
+// CHECK-NEXT:   addq (%rip), %r28
+// CHECK-NEXT:   R_X86_64_CODE_4_GOTTPOFF    tls0-0x4
+// CHECK-NEXT:   addq (%rip), %r16
+// CHECK-NEXT:   R_X86_64_CODE_4_GOTTPOFF    tls1-0x4
+
+.type tls0,@object
+.section .tbss,"awT",@nobits
+.globl tls0
```

Comment:
drop `.align` and `.size`, unneeded by this test

Reply:
Done.